README.md

```diff
@@ -2,24 +2,75 @@
 
 Because everybody wants a pony.
```

Comment:
Meaningful tagline as introduction:

Pony is a gRPC [service?] fake.

Reply:
done